### ISSUE TITLE
Reproducing top 1 accuracy in Figure 4

### ISSUE THREAD

**Issue by mlpen** [bomri/SlowFast#7]
Hi, 

I am trying to reproduce the top 1 accuracy of ViT-B-VTN in Figure 4. I trained a model using the config file in `configs/Kinetics/VIT_B_VTN.yaml`, but I only get 68.5% accuracy after epoch 9. The following is the train and val result from my experiment. Is there anything that I have overlooked? Thanks! 

train: 
```
epoch
1/25    41.12769
2/25    54.65638
3/25    58.83401
4/25    61.77271
5/25    63.87714
6/25    65.69931
7/25    67.11017
8/25    68.33328
9/25    69.61342
```

val:
```
epoch
2/25    59.21026
3/25    62.00704
4/25    64.85412
5/25    65.65895
6/25    67.18813
7/25    67.39437
8/25    68.49095
9/25    68.50604
```

**Comment by bomri** [comment]
Hi @mlpen,
Yes, this seems a bit low for epoch 9.
Does your dataset is full and properly encoded?
See this page for [Dataset Preparation](https://github.com/facebookresearch/video-nonlocal-net/blob/main/DATASET.md)

**Comment by mlpen** [comment]
Thanks for replying. I downloaded the dataset according to this [repo](https://github.com/cvdfoundation/kinetics-dataset) and then follow the instructions from [slowfast](https://github.com/facebookresearch/SlowFast/blob/main/slowfast/datasets/DATASET.md) to prepare the dataset.

**Comment by bomri** [comment]
How many videos do you have in your train and validation sets?

**Comment by mlpen** [comment]
240214 in train set and 19879 in val set.

**Comment by bomri** [comment]
OK, those are not exactly the same numbers we have but close enough.

You can try and use our pre-trained [model weights](https://github.com/bomri/SlowFast/blob/master/MODEL_ZOO.md#vtn-model-details-in-projectsvtn) and only run evaluation on the valid set. It can help understand if the gap is related to data or something else.

**Comment by mlpen** [comment]
Ok, I will try the pre-trained weights. After the model training completed, the final val accuracy is 76.3 and train accuracy is 88.5.
